app/league/[leagueId]/entry/[entryId]/week/WeekTeams.tsx

```diff
@@ -7,27 +7,69 @@ import { RadioGroup } from '@radix-ui/react-radio-group';
 import { IWeekTeamsProps } from './WeekTeams.interface';
 import { WeeklyPickButton } from '@/components/WeeklyPickButton/WeeklyPickButton';
 
+/**
+ * Formats the date to 'day, mon date' format and the time to either 12 or 24-hour format based on the user's locale.
+ * @param dateStr The date string to format.
+ * @returns The formatted date and time string.
+ */
+const formatDateTime = (dateStr: string): string => {
```

Comment:
Do not shorthand your variable names. When someone else is reading them, you want them to concise and ready to browse.
```suggestion
const formatDateTime = (dateString: string): string => {
```